consensus/types/src/chain_spec.rs

```diff
@@ -136,6 +136,7 @@ pub struct ChainSpec {
     pub maximum_gossip_clock_disparity_millis: u64,
     pub target_aggregators_per_committee: u64,
     pub attestation_subnet_count: u64,
+    pub sync_committee_subnet_count: u64,
```

Comment:
I'm not seeing this used anywhere, only seeing `consts::altair::SYNC_COMMITTEE_SUBNET_COUNT` used